configs/components/ruby-2.1.9.rb

```diff
@@ -182,6 +182,11 @@
     rbconfig_changes["CC"] = "x86_64-w64-mingw32-gcc"
   end
 
+  pkg.add_source("file://resources/files/rubygems/COMODO_RSA_Certification_Authority.pem")
+  defaults_dir = File.join(settings[:libdir], "ruby/2.5.0/rubygems/ssl_certs/puppetlabs.net")
```

Comment:
I think this should be `ruby/2.1.0` instead of `ruby/2.5.0`

Reply:
puff, copy/paste ...